bin/backstop

```diff
@@ -20,19 +20,28 @@ ROOT_DIR=$(dirname "$BIN_DIR")
 VERSION=6.1.1
 IMAGE=googlesitekit/vrt:$VERSION
 
-# Build the backstopjs image if it doesn't exist yet.
-docker image inspect $IMAGE -f "{{.Id}}" >/dev/null 2>&1 ||
-	docker build -t $IMAGE --build-arg BACKSTOPJS_VERSION=$VERSION "$ROOT_DIR/docker/backstopjs"
+# (re)Build the backstopjs image as needed.
+docker build \
```

Comment:
Docker has its own caching, so this check isn't necessary as it only adds ~1sec or so if cached.